dist/aphrodite.js

```diff
@@ -218,6 +218,9 @@ module.exports =
 	 */
 	var isUnitlessNumber = {
 	    animationIterationCount: true,
+	    borderImageOutset: true,
```

Comment:
Oh yea I agree. I typically trust the gitignore to handle that though but I can revert that unless you find time too. 

> On Apr 17, 2016, at 11:17 AM, Kent C. Dodds <notifications@github.com> wrote:
> 
> In dist/aphrodite.js:
> 
> > @@ -218,6 +218,9 @@ module.exports =
> >  	 */
> >  	var isUnitlessNumber = {
> >  	    animationIterationCount: true,
> > +	    borderImageOutset: true,
> It's because you ran the build and committed the dist directory. This is why I recommend not committing generated files: “Why I don’t commit generated files to master” @kentcdodds https://medium.com/@kentcdodds/why-i-don-t-commit-generated-files-to-master-a4d76382564
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>